iac/main.tf

```diff
@@ -15,7 +15,15 @@ terraform {
 
 # Configure the AWS Provider
 provider "aws" {
-  region = "us-east-1"
+  region = var.aws_region
```

Comment:
Adding default tags at the provider level is excellent for governance and cost tracking.